uart/uart_posix.c

```diff
@@ -32,6 +33,10 @@
  * proxmark3 project.
  */
 
+#define __USE_MISC
```

Comment:
You never should define one of the `__USE` defines.

Reply:
Fixed